### ISSUE TITLE
<button> should allow for attributes

### ISSUE THREAD

**Issue by cossssmin** [foundation/foundation-emails#592]
**How can we reproduce this bug?**

```
<button href="#" class="large" align="right">Button Label</button>
```

**What did you expect to happen?**

Since `<button>` generates a table, I would expect it to have the `align="right"` attribute, just like `<columns>` do.

**What happened instead?**

The generated table does not contained the passed attribute.

**What email clients does this happen in?**

Viewing HTML source, no need to email client testing.

This would be useful for <strike>aligning buttons on their own, no need to complicate things with columns.</strike> not just alignment, any HTML or custom attributes one might need to pass.

**Comment by cossssmin** [comment]
Looks like someone already created a PR, almost 3 months ago: @yvmarques, https://github.com/zurb/inky/pull/61

**Comment by rafibomb** [comment]
Merged zurb/inky#61
